cdap-cli/src/main/java/co/cask/cdap/cli/util/ArgumentParser.java

```diff
@@ -61,6 +61,30 @@
   }
 
   /**
+   * Parses a map in the format: "key1=a key2=b .."
+   *
+   * @param mapString {@link String} representation of the map
+   * @return the map
+   */
+  public static Map<String, Integer> parseStringIntegerMap(String mapString) {
```

Comment:
I think this method can be replaced with 1liner:
`return Splitter.on(" ").withKeyValueSeparator("=").split(mapString);`